bench/README.md

```diff
@@ -4,11 +4,12 @@
 This folder contains two Haskell programs that work together to simplify the
 performance analysis of ghcide:
 
-- `exe/Main.hs` - a standalone benchmark suite. Run with `stack bench`
+- `exe/Main.hs` - a standalone benchmark runner. Run with `stack exec ghcide-bench`
```

Comment:
Do you mean `stack run ghcide-bench`? `stack-exec` would require it building first.

Reply:
Oops!